src/core.rs

```diff
@@ -800,6 +831,7 @@ impl Core {
     fn dispatch_request_response(&mut self,
                                  routing_msg: RoutingMessage)
                                  -> Result<(), RoutingError> {
+        trace!("{:?}Handling - {:?}", self, routing_msg);
```

Comment:
I wouldn't reintroduce those trace statements: We log most of the requests and responses in the handling methods.
(Or was that a git rebase gone wrong?)